### DISCUSSION TITLE
Change Keybindings (Thonny 4.1.4)

### DISCUSSION THREAD

**Discussion by silverleaf21** [thonny/thonny#3263] [General]
Hi,
I'm using Thonny on Linux and Windows for Python/Micropython programming.
Is it possible to manually change the keybinding for certain functions? Like is there an option or keybindings settings file to set a custom keybinding for toggling comments? I really get used to the ctlr+shift+7 from another IDE (used for different non python projects).

I found this [issue](https://github.com/thonny/thonny/issues/2343) about vim keybindings. But that is a bit older, so things might have changed. I think more of "Let a user customize some individual keybindings and leave others default thonny" and not "Have a setting where I can choose a preconfigured keybinding style from another program".

**Comment by Hermann12** [comment]
https://github.com/thonny/thonny/wiki/Custom-shortcuts

**Reply by silverleaf21** [reply]
Thanks, I only searched for keybindings... My fault.
